app/app.go

```diff
@@ -82,6 +83,12 @@ func (app *karlsendApp) main(startedChan chan<- struct{}) error {
 
 	// Show version at startup.
 	log.Infof("Version %s", version.Version())
+	log.Infof("Using KarlsenHashV2 impl: %s", pow.GetHashingAlgoVersion())
+	if !app.cfg.Testnet && !app.cfg.Devnet && !app.cfg.Simnet {
```

Comment:
Ok for me and very good that builds for `testnet-1` are not allowed to connect to `mainnnet`.